### DISCUSSION TITLE
APR-DRG Severity of Illness (SOI) - How is this being handled in the In-Network Rate File

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#190] [Q&A]
How should the APR-DRG SOI values be handled in the In-Network Rate File?   Should the value of the APR-DRG have reference to the SOI value?  For example,  Billing Code:  APR-DRG  Billing Code Value 001-1, 001-2, 001-3, 001-4, etc?   or if only 001 is used, then are all rates just published under the one code with no SOI reference?

**Comment by reevesj2** [comment]
Is there any update from CMS?  If not, what are other companies planning to do?

**Comment by tdfow** [comment]
@shaselton   any updates?

**Comment by ken-talonhealthtech** [comment]
I would ask the same for MS-DRG Classifications (e.g. MS-DRG 885)

**Comment by tdfow** [comment]
@shaselton @shaselton-usds Did CMS ever publish a response to this discussion item under a different thread?  I am not seeing anything on this.  Can we get a CMS response on this?

**Comment by shaselton-usds** [comment]
@tdfow Absent a specific attribute to capture SOIs, could that information be included in the additional notes? That would allow for the distinguishing of multiple negotiated rates

**Reply by nmayle** [reply]
Did this ever get resolved? The severity level appears to be treated by most as just one of the digits of the APR-DRG code.

In any event, since the SOI changes how much is paid (as the equivalent does in MS-DRGs) without the digit describing the severity of illness subclass, the APR-DRG code would not meet this description in the rule:

> The final rules require plans and issuers to include in the machine-readable files a billing code or other code used to identify covered items or services for purposes of claims adjudication, payment, and cost-sharing liability
